apps/angular/src/app/app.component.html

```diff
@@ -1 +1,7 @@
+<ng-container *ngIf="isAuthenticated$ | async">
+    <header class="app-header">
+        <button type="button" mat-button (click)="handleLogout()">Logout</button>
+    </header>
+</ng-container>
+
 <router-outlet></router-outlet>
```

Comment:
Let's use self closing tags for Angular components where possible:
```suggestion
<router-outlet />
```